Content.Client/SurveillanceCamera/UI/SurveillanceCameraMonitorWindow.xaml.cs

```diff
@@ -125,14 +125,12 @@ public void UpdateState(IEye? eye, HashSet<string> subnets, string activeAddress
 
     private void PopulateCameraList(Dictionary<string, string> cameras)
     {
-        SubnetList.Clear();
-
-        foreach (var (address, name) in cameras)
-        {
-            AddCameraToList(name, address);
-        }
-
-        SubnetList.SortItemsByText();
+        var entries = cameras.Select(i => new ItemList.Item(SubnetList) {
+            Text = $"{i.Value}: {i.Key}",
+            Metadata = i.Key
+        }).ToList();
+        entries.Sort((a, b) => string.Compare(a.Text, b.Text, StringComparison.Ordinal));
+        SubnetList.SetItems(entries, (a,b) => string.Compare(a.Text, b.Text));
```

Comment:
'SetItems' is popping up as a missing method in my IDE, and nothing else in the current code base seems to be using it. Please check this and update if necessary

Reply:
This PR depends on https://github.com/space-wizards/RobustToolbox/pull/5425

If preferable, I can try making SetItems an extension method in Content.Client, so it doesn't require an RT change.

Reply:
Whoops, not sure how I managed to miss that!

As a heads up, we only have a few RT maintainers around right now, it could be a while before an engine PR gets merged